samples/MvcSandbox/Controllers/HomeController.cs

```diff
@@ -14,5 +14,9 @@ public IActionResult Index()
         {
             return View();
         }
+
+        [HttpGet("")]
+        [ProblemDescription]
```

Comment:
The spec (https://tools.ietf.org/html/rfc7807) uses the term "problem *details*" rather than "problem description". Might a different name like `[ReturnsProblemDetails]` be more descriptive?

Reply:
That's a good point. We might want to rename that class.

Reply:
The attribute that's here is just an example. 

My plan is for us to introduce an `[ApiController]` attribute, and mostly have the 'problem filter' hidden from users